modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/sequence/steps/StepsTable.scala

```diff
@@ -1071,9 +1077,10 @@ object StepsTable extends Columns {
       TableContainer.Props(
         b.props.hasControls,
         size => {
+          val areaSize = Size(size.height, size.width - 15) // Subtract scroll bar width
```

Comment:
Where does the 15 come from? It feels it would be browser/OS dependent

Reply:
Actually, it is. Let me add some code to detect the browser.